docs/README.md

```diff
@@ -24,6 +24,7 @@ k0s is distributed as a single binary with zero host OS dependencies besides the
 - Supports custom [Container Network Interface (CNI)](networking.md) plugins (Kube-Router is the default, Calico is offered as a preconfigured alternative)
 - Supports custom [Container Runtime Interface (CRI)](runtime.md) plugins (containerd is the default)
 - Supports all Kubernetes storage options with [Container Storage Interface (CSI)](storage.md)
+    - Bundled with [host-local storage provider](storage.md#bundled-openebs-storage)
```

Comment:
Alternatively it could be like this, in one line:
- Supports all Kubernetes storage options with [Container Storage Interface (CSI)](storage.md#csi), includes [OpenEBS local storage](storage.md#bundled-openebs-storage).

Reply:
fixed